prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -489,6 +490,7 @@ def prompt_processor() -> Any:
                                 **usage_kwargs,
                                 PSKeys.LLM_USAGE_REASON: PSKeys.CHALLENGE,
                             },
+                            capture_metrics=True,
```

Comment:
Have a genera qn. Is the logic like: when include_metrics=true from API, we will pass capture_metrics = true and only then SDK genertes the metrics?

Reply:
@gaya3-zipstack 

No.
Earlier it was like that for include_metadata, now it is changed.
Now we always calculate metrics and metadata.
And remove the keys before sending it as response for API based on values sent on form-data

Reply:
Ok. After the call, I understand the design issues that we have around this and why are are forced to send the boolean always as true disregarding the user request. When response comes, only then we go back and remove the metadata/metrics if user did not want it. Obviously better design would be to honour the flag at all times and constrict our computation accordingly. This is not a quick thing to get done. But let us consider this as a design improvement for future.